index.bs

```diff
@@ -290,32 +284,250 @@ Loader instances are initially created with the internal slots described in the
 
 <h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
 
-<h4 id="registry-GetCurrentStage" aoid="GetCurrentStage">GetCurrentStage(entry)</h4>
+<h4 id="registry-CreateRegistry" aoid="CreateRegistry">CreateRegistry()</h4>
+
+The abstract operation CreateRegistry with no arguments performs the following steps:
+
+<emu-alg>
+1. Let _O_ be ? OrdinaryCreateFromConstructor(Registry, "%RegistryPrototype%", «[[RegistryMap]]» ).
+1. Let _M_ be ObjectCreate(%MapIteratorPrototype%, «[[Map]], [[MapNextIndex]], [[MapIterationKind]]»).
+1. Set _O_’s [[RegistryMap]] internal slot to _M_.
+1. Return _O_.
+</emu-alg>
+
+<h3 id="registry-constructor">The Registry Constructor</h3>
+
+The Registry constructor is the %Registry% intrinsic object. When called as a constructor it creates and initializes a new Registry object. When <b>Registry</b> is called as a function rather than as a constructor, it throws an exception.
```

Comment:
This is still inconsistent with Registry.prototype.constructor. As I said before, it should be the case that `Registry === Registry.prototype.constructor`.